### PR TITLE
RESTWS-837: Add documentation for VisitsConfigurationController2_0

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Related PR: https://github.com/openmrs/openmrs-module-webservices.rest/pull/489
Related issue: https://issues.openmrs.org/browse/RESTWS-837

### CODE REVIEW

File: source/includes/Visit/visits_configuration.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Visits Configuration
+
+## Visits Configuration Overview
+
+Visits introduce set of settings related to Visits Behavior. You can retrieve and update this configuration through REST API.
```
Comment (dkayiwa) [inline source/includes/Visit/visits_configuration.md]: Can you correct the grammar above?

Comment (BartlomiejRasztabiga) [reply source/includes/Visit/visits_configuration.md]: I'm not sure where is the grammar error. Could you point it out?

Comment (dkayiwa) [reply source/includes/Visit/visits_configuration.md]: After `introduce`

Comment (BartlomiejRasztabiga) [reply source/includes/Visit/visits_configuration.md]: I meant set of settings, that is multiple settings :)

Comment (dkayiwa) [reply source/includes/Visit/visits_configuration.md]: a set of settings

Comment (BartlomiejRasztabiga) [reply source/includes/Visit/visits_configuration.md]: Resolved here: https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/151/commits/34e26613f5d59ae674dcc197a3c094cb2448c576

**Pull request merged**